tools/plotjuggler/juggle.py

```diff
@@ -66,12 +66,12 @@ def start_juggler(fn=None, dbc=None, layout=None):
 
   extra_args = ""
   if fn is not None:
-    extra_args += f"-d {fn}"
+    extra_args += f" -d {fn}"
   if layout is not None:
-    extra_args += f"-l {layout}"
+    extra_args += f" -l {layout}"
 
-  subprocess.call(f'plotjuggler --plugin_folders {INSTALL_DIR} {extra_args}',
-                  shell=True, env=env, cwd=juggle_dir)
+  cmd = f'plotjuggler --plugin_folders {INSTALL_DIR}{extra_args}'
```

Comment:
why remove the space between INSTALL_DIR and extra_args?

Reply:
because then we have a trailing space with no extra params and two spaces before the first param